src/OpenTelemetry.Exporter.Prometheus.HTTP.Server/PrometheusExporterHttpServerOptions.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="PrometheusExporterHttpServerOptions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using OpenTelemetry.Exporter.Prometheus.HTTP.Server;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// <see cref="PrometheusExporterHttpServer"/> options.
+    /// </summary>
+    public class PrometheusExporterHttpServerOptions
+    {
+        internal const string DefaultScrapeEndpointPath = "/metrics";
+        private IReadOnlyCollection<string> httpListenerPrefixes = new string[] { "http://localhost:9464/" };
+
+#if NETCOREAPP3_1_OR_GREATER
+        /// <summary>
+        /// Gets or sets a value indicating whether or not an http listener
+        /// should be started. Default value: False.
+        /// </summary>
+        public bool StartHttpListener { get; set; }
```

Comment:
Let's remove the `StartHttpListener` option and always start it. This was only needed before because the one project had multiple ways to do it. Now if you don't want to use the listener you just wouldn't reference the package 😄

Reply:
That's what I have thought but I was not entirely sure whether to remove it at the time. Thank you for confirming it!